libraries/credential-cache/src/CredentialCache.ts

```diff
@@ -79,7 +76,7 @@ export class CredentialCache /* implements IDisposable */ {
       cacheDirectory = path.dirname(options.cacheFilePath);
       cacheFileName = options.cacheFilePath.slice(cacheDirectory.length + 1);
     } else {
-      cacheDirectory = RushUserConfiguration.getRushUserFolderPath();
+      cacheDirectory = `${User.getHomeFolder()}/.rush-user`;
```

Comment:
Is `.rush-user` a const in `rush-lib`? If it is, this package should probably take a devDependency on that and use a `const RUSH_USER_FOLDER_NAME: typeof import('@microsoft/rush-lib').Constants.RushUserFolderName = '.rush-user';` constant.

Reply:
This package cannot take a `devDependency` on rush-lib because rush-lib has a runtime dependency on this package. I could, however, move the real definition of this constant into credential-cache and clone/reexport it.